.scalafmt.conf

```diff
@@ -8,4 +8,5 @@ rewrite.rules = [ AvoidInfix, ExpandImportSelectors, RedundantParens, SortModifi
 rewrite.sortModifiers.order = [ "private", "protected", "final", "sealed", "abstract", "implicit", "override", "lazy" ]
 spaces.inImportCurlyBraces = true   # more idiomatic to include whitepsace in import x.{ yyy }
 trailingCommas = preserve
+newlines.afterCurlyLambda = preserve
```

Comment:
Added this new rule to prevent a rather large code change. We can discuss in other PRs whether we still want this setting or not.